Resources/public/css/layout.css

```diff
@@ -29,6 +29,7 @@
     opacity: 1;
     min-height: 100%;
     overflow: auto;
+    pointer-events: all;
```

Comment:
> Well, where is it placed doesn't really matter. The is-app-open CSS class is added as soon as the Loading application screen disappears.

for you maybe, for me (as a maintainer of PlatformUI) it matters. The opposite of 'is-app-loading' is not having 'is-app-loading', not 'is-app-open' which is a different concept.

> We can keep the .is-app-loading selector alone without .is-app-open at the beggining of the selector.

so please do NOT add unneeded changes. A bug fix is a fix with the minimum amount of change.

> I did it this way to show, that .is-app-loading class is added when the app is opened but switching active views.

which is an implementation detail irrelevant to the problem. *open* and *loading* are 2 different concepts which are not really related.

> To sumarize, I'd keep it here as it is.

I prefer to not answer to that...